### PR TITLE
OP-1068 - Script consolidation update

### PR DESCRIPTION
**Pull request by mizzioisf**
Update 23.05: reviewed restore and install functions after reviews.
Needs one last good run of testing and then finally should be mergeable :-)

-> Separated clean and kill functions
- Updated scripts with included **oh-api** functionalities
- Reviewed interactive menu: select "E" for Expert mode and show advanced options
- Updated minor Java version
- Bugfixes

See https://openhospital.atlassian.net/browse/OP-1068

### CODE REVIEW

File: oh.sh [hunk 1]
```diff
@@ -768,6 +830,51 @@ function test_database_connection {
...
+		exit 1;
+	fi
+
+	echo "------------------------"
+	echo "---- EXPERIMENTAL ------"
+	echo "------------------------"
+	echo "Starting API server..."
+	echo "Please wait, it might take some time..."
+	echo ""
+	echo "Connect to http://$OH_UI_URL:8080 for dashboard"
```
[the remark this answers is not in the document]
Comment (mizzioisf) [reply oh.sh:847]: Yes ! Thank you !

File: oh.ps1 [hunk 2]
```diff
@@ -1255,8 +1255,14 @@ if ( $INTERACTIVE_MODE -eq "on" ) {
 			initialize_dir_structure;
 			set_language;
 			mysql_check;
+			Write-Host "Do you want to create the [$DATABASE_USER] user and [$DATABASE_NAME] database on [$DATABASE_SERVER] server ?"
```
Comment (dbmalkovsky) [inline oh.ps1:1262]:
```suggestion
			Write-Host "Do you want to create the [$DATABASE_USER] user and [$DATABASE_NAME] database on [$DATABASE_SERVER] server?"
```

File: oh.sh [hunk 3]
```diff
@@ -1121,8 +1121,15 @@ function parse_user_input {
 		initialize_dir_structure;
 		set_language;
 		mysql_check;
+		echo "Do you want to create the [$DATABASE_USER] user and [$DATABASE_NAME] database on [$DATABASE_SERVER] server ?"
```
Comment (dbmalkovsky) [inline oh.sh:1129]:
```suggestion
		echo "Do you want to create the [$DATABASE_USER] user and [$DATABASE_NAME] database on [$DATABASE_SERVER] server?"
```

Comment (mizzioisf) [reply oh.sh:1129]: Fixed, will update PR, thank you !

**Pull request merged**